src/stylesheets/components/_advantage.scss

```diff
@@ -1,3 +1,25 @@
+$title-font-size: 20px;
+
 .advantage {
-  
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+
+  &__icon {
+    width: 70px;
+  }
+
+  &__heading {
+    font-size: $title-font-size;
+    font-weight: 600;
+    margin-top: 30px;
+    max-width: 260px;
+  }
+
+  &__description {
+    margin-top: 20px;
```

Comment:
I prefer use `padding-top` instead of `margin-top` because margin has a weird issue with overlapping. There is an article about it on Css-Tricks.